taskcluster/docker/nimbledroid/Dockerfile

```diff
@@ -0,0 +1,6 @@
+# %ARG DOCKER_IMAGE_PARENT
+FROM $DOCKER_IMAGE_PARENT
+
+MAINTAINER Kate Glazko <kglazko@mozilla.com>
```

Comment:
@kglazko I left your name here because I remember you own nimbledroid. Please let me know what you think 🙂